--- a/webapp/graphite/settings.py
+++ b/webapp/graphite/settings.py
@@ -34,7 +34,7 @@
 INDEX_FILE = STORAGE_DIR + 'index'
 WHITELIST_FILE = LISTS_DIR + 'whitelist'
 LOG_DIR = STORAGE_DIR + 'log/webapp/'
-
+CARBON_DIR = '/var/lib/carbon/'
 CLUSTER_SERVERS = []
 
 sys.path.insert(0, THIRDPARTY_DIR)
@@ -44,8 +44,8 @@
 # local_settings.py, instead you should override DATA_DIRS
 # to list all directories that should be searched for files
 # of a supported format.
-WHISPER_DIR = STORAGE_DIR + 'whisper/'
-RRD_DIR = STORAGE_DIR + 'rrd/'
+WHISPER_DIR = CARBON_DIR + 'whisper/'
+RRD_DIR = CARBON_DIR + 'rrd/'
 try:
   import rrdtool
   DATA_DIRS = [WHISPER_DIR, RRD_DIR]
