Index: graphite-web/webapp/graphite/settings.py
===================================================================
--- graphite-web.orig/webapp/graphite/settings.py	2012-06-15 12:38:46.000000000 +0200
+++ graphite-web/webapp/graphite/settings.py	2012-06-15 13:14:54.000000000 +0200
@@ -40,6 +40,7 @@
 DASHBOARD_CONF = ''
 GRAPHTEMPLATES_CONF = ''
 STORAGE_DIR = ''
+CARBON_DIR = '/var/lib/carbon'
 WHITELIST_FILE = ''
 INDEX_FILE = ''
 LOG_DIR = ''
@@ -139,7 +140,7 @@
 if not LOG_DIR:
   LOG_DIR = join(STORAGE_DIR, 'log', 'webapp')
 if not WHISPER_DIR:
-  WHISPER_DIR = join(STORAGE_DIR, 'whisper/')
+  WHISPER_DIR = join(CARBON_DIR, 'whisper/')
 if not RRD_DIR:
   RRD_DIR = join(STORAGE_DIR, 'rrd/')
 if not DATA_DIRS:
